spec/pandoc_parser_spec.rb

```diff
@@ -1,63 +1,35 @@
 require 'spec_helper'
+require 'pandoc-ruby'
 
-describe Jekyll::Converters::Markdown::PandocParser do
+describe Jekyll::Converters::Markdown::PandocProcessor do
 
-  let(:config) {{ 'markdown' => 'pandoc',
-                  'pandoc' => { 'extensions' => ["smart", "mathjax"] }}}
-  let(:markdown) { "## This is a heading" }
-  let(:html) { "<h2 id=\"this-is-a-heading\">This is a heading</h2>\n" }
 
-  describe "pandoc-ruby" do
-    subject { PandocRuby }
-
-    it "should convert markdown" do
-      subject.new(markdown).to_html.should eq(html)
+  describe 'pandoc' do
+    config = {'markdown' => 'pandoc'}
+    config = Jekyll::Configuration::DEFAULTS.merge(config)
+    subject { Jekyll::Converters::Markdown::PandocProcessor.new config }
+    it 'converts markdown' do 
+      markdown = "## This is a heading"
+      html = "<h2 id=\"this-is-a-heading\">This is a heading</h2>\n"
+      expect(subject.convert(markdown)).to eq(html)
     end
   end
 
-  describe "class methods" do
-    subject { Jekyll::Converters::Markdown::PandocParser }
-
-    it "should initialize" do
-      parser = subject.new(Jekyll::Configuration::DEFAULTS.merge(config))
-      parser.public_methods.should include(:convert, :config_option)
+  describe 'bibliography' do
+    config = {
+      'markdown' => 'pandoc',
+      'pandoc' => {
+        'extensions' => {
+          'bibliography' => 'spec/citations.bib'
+	}
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.<br>Tab detected.